data/human/ships.txt

```diff
@@ -1565,6 +1565,57 @@ ship "Dropship"
 
 
 
+ship "Enforcer"
+	sprite "ship/enforcer"
+	thumbnail "thumbnail/enforcer"
+	attributes
+		category "Light Warship"
+		"cost" 490000
+		"shields" 2300
+		"hull" 1100
+		"required crew" 3
+		"bunks" 5
+		"mass" 130
+		"drag" 2.9
+		"heat dissipation" .7
+		"fuel capacity" 400
+		"cargo space" 30
+		"outfit space" 210
+		"weapon capacity" 55
+		"engine capacity" 60
+		weapon
+			"blast radius" 34
+			"shield damage" 340
+			"hull damage" 170
+			"hit force" 510
+	outfits
+		"Energy Blaster" 4
+		"Laser Turret"
+		
+		"nGVF-CC Fuel Cell"
+		"LP036a Battery Pack"
+		"D14-RN Shield Generator"
+		
+		"X2700 Ion Thruster"
+		"X2200 Ion Steering"
+		"Hyperdrive"
+
+	engine 0 39
+	gun -11 -52
+	gun 11 -52
+	gun -5.5 -48
+	gun 5.5 -48
+	turret 0 -0.5
+	leak "leak" 40 30
+	leak "flame" 50 60
+	explode "tiny explosion" 8
+	explode "small explosion" 12
+	explode "medium explosion" 4
+	description "Prior to the formation of the galactic Republic, many star sectors were owned by small governing corporate entities, with the vast majority of space being inaccessible to private citizens. The only exception to this rule were the pirate bands, whose few ships were often stolen or rebellious corporate fleets of their own, yet who had an outsized impact on interstellar trade. The necessity for corporations to protect their operations meant that Enforcer was one of the more common system security vessels used for defense. The ship boasts an armament of four gun ports and a turreted mount, all packed on a small but defensible frame that was powerful for its time."
+	description "	Although no longer on the market, the Enforcer finds itself in the possession of a small number of merchants, or, ironically, various groups of pirates, as an aging combat ship easily outperformed by other light warships or even interceptors of a similar size."
```

Comment:
I am reviewing one more time so I can approve one more time.
```suggestion
	description `	Although no longer on the market, the Enforcer finds itself in the possession of a small number of merchants and, ironically, various groups of pirates, as an aging combat ship easily outperformed by other light warships or even interceptors of a similar size.`
```
Is the description is saying the Enforcer is only a merchant ship or only a pirate ship but not both? Seems conflicting.

Comma looks funny if you remove "ironically": `Although no longer on the market, the Enforcer finds itself in the possession of a small number of merchants, or various groups of pirates`